AggregatedServices/Model/Service/GetCart.php

```diff
@@ -164,11 +182,35 @@ public function get(
             ->create();
         $aggregatedCart->setProducts($this->productRepository->getList($productsSearchCriteria));
 
-        // Fetch cart products
+        // Fetch attributes metadata
         if ($productAttributesSearchCriteria) {
```

Comment:
Let's at least move it to a separate method to make code cleaner

Reply:
Resolved